container/cli.py

```diff
@@ -264,14 +260,19 @@ def __call__(self):
             LOGGING['loggers']['container']['level'] = 'DEBUG'
         config.dictConfig(LOGGING)
 
+        vargs = vars(args)
+        if not vargs['project_name']:
```

Comment:
We could add `project_name` to settings, yes. However, that's not the point of this change. The point here is to reduce redundant code in `core.py`.